update/update-version.md

```diff
@@ -1,139 +1,187 @@
-        - In the left navigation column, click on **VPC Infrastructure** > **VPCs**.
-        - Select the region in which the VPC of the VSI was deployed.
-        - Select the VPC associated with the VSI deployment.
-        - Copy the **VPC ID** from the Overview tab.
-
-1. [Optional] If the VSIs are being deployed in an account that is different from the account where the Schematics Workspace are located, you need to provide an IBM Cloud API key, which can be used for fetching the VPC details. In this case, you can pass the VPC API-KEY to the script using `-k` flag. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
+1. Get the VPC IDs and the regions of any VPCs that the VSIs are deployed in by using the IBM Cloud console:
+    1. Click the 3-layer menu icon, and then click **VPC Infrastructure** > **VPCs**.
+    1. Select the region in which the VPC of the VSI was deployed.
+    1. Select the VPC that is associated with the VSI deployment.
+    1. Copy the **VPC ID**.
+1. Run the script:
 
-1. Run the script from your local machine
     ```sh
     bash schematics_update_v3.x.x_to_v4.x.x.sh -v "<vpc-id1>[,<vpc-id2>,...]" -r "<vpc-region>" [-k "<vpc-ibm-api-key>"]
     ```
-    This script will trigger a new job in the schematics workspace, please monitor the state of the job in IBM Cloud UI. Check if the job completes **Successfully**, in that case go ahead with **Step 7**. In case of schematics workspace job ends in a failed state, please follow **Step 9** to revert any changes made by the script.
 
-1. The user would have to update their code to consume version 4.x.x, and then update their schematics workspace to the version of their code that contains the updated module and click on `Generate plan` and make sure none of the VSIs will be recreated. The expected behaviour should be that the plans contains only naming updates, and no destroy / re-creates of any resources.
+    - Replace `<vpc-id1>` and `<vpc-region>` with the information that you copied earlier.
+    - If the VSIs are deployed in an account that is different from the account where the Schematics workspace is located, include the `-k` option and replace `<vpc-ibm-api-key>` with the IBM Cloud API key with access to the VPC.
 
-1. Click on `Apply plan`.
+    The script creates a job in the Schematics workspace.
 
-1. [Optional] If a schematics workspace job that was initiated through the script encounters an issue, you can undo any modifications made by running the script again with the -z flag. For example,
-    ```sh
-    bash schematics_update_v3.x.x_to_v4.x.x.sh -z
-    ```
-    A new schematics workspace job reverting the state back to its prior condition, which existed prior to the execution of the script. The script will use the `revert.json` file that got created when the script initially ran
+1.  Monitor the status of the job in the IBM Cloud [console](https://cloud.ibm.com/schematics/workspaces).
```

Comment:
maybe it should say IBM Cloud Schematics workspace console?

Reply:
changed to `Schematics workspaces dashboard`